src/reference-designs/tko-on-vsphere-with-tanzu.md

```diff
@@ -310,10 +310,13 @@ The initial configuration and setup of the platform does not require any externa
 
 When vSphere with Tanzu is deployed on VDS networking, you can deploy an external container registry (Harbor) for Tanzu Kubernetes clusters.
 
-There are two main supported installation methods for Harbor:
+There are three officially documented installation methods for Harbor:
 
-*   [Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster: This installation method may be preferred if you are already invested in Helm.
-*   [VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs) using `docker-compose`: This installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor.
+
+*   [**Tanzu Kubernetes Grid Package deployment**](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-packages-harbor-registry.html) on a Tanzu Kubernetes Grid shared services cluster cluster - Tanzu Kubernetes Grid includes signed binaries for Harbor, which you can deploy into a shared services cluster to provide container registry services for other Tanzu Kubernetes (workload) clusters. This installation method is recommended for general use cases.
+air-gapped or internet-less deployments and must not be used for hosting application images.
+*   [Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster: This installation method may be preferred if you are already invested in Helm. Helm deployments are only supported by the Open Source community and not by VMware Global Support Services.
+*   [VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs) using `docker-compose`: This installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor. VM-based deployments are only supported by VMware Global Support Services to host the system images for 
```

Comment:
Replace the last sentence with the following:
In an air-gapped or Internet-less deployment, VMware Global Support Services only supports hosting system images on VM-based deployments. Hosting application images is not supported.

Reply:
This should be addressed in the latest push